recipes/CRISPRStudio/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mkdir -p $PREFIX/bin/tools/
+cp CRISPR_Studio_1.0.py $PREFIX/bin
+cp -r fasta-36.3.8g $PREFIX/bin/tools
+ln -s  $PREFIX/bin/CRISPR_Studio_1.0.py $PREFIX/bin/CRISPR_Studio
```

Comment:
Why not just rename the script?

Reply:
I thought it needed to be soft linked I guess not. Changed.